src/orderbook.js

```diff
@@ -164,7 +201,15 @@ OrderBook.EVENTS = [
   'offer_changed', 'offer_funds_changed'
 ];
 
-OrderBook.DEFAULT_TRANSFER_RATE = 1000000000;
+OrderBook.DEFAULT_TRANSFER_RATE = Amount.from_json(1000000000);
+
+OrderBook.NOTIFY_THROTTLE_TIME = 4000;
```

Comment:
No definitely debounce computeAutobridgedOffersWrapper - having both legs modified in one ledger happens quite regularly.  right now I have `notify` at timeout = 100, maxWait at 500, and `compute` at 250/500 and it seems to work very well with your other optimizations.  I don't have `compute` adjusting dynamically

I think we should go with something like that, and get it into ripple trade and charts, and then consider further optimization.  This is already phenomenal improvement.